### ISSUE TITLE
Plantear el Problema

### ISSUE THREAD

**Issue by Karina624** [KhemSearch/OpenKhem#3]

**Comment by LauraCarbonell** [comment]
![abejas](https://user-images.githubusercontent.com/32964822/32457689-4794246a-c32a-11e7-86a8-3215f3b53dd5.jpg)

He encontrado este problema de actualidad, ¿que os parece si profundizamos en este tema ?

**Comment by Karina624** [comment]
Podríamos trabajar sobre este problema, sin polinización el ser humano corre peligro :(

**Comment by marthapd** [comment]
Ya tenemos problema.  Tenemos que centrarnos en los efectos sobre el Cambio Climático. Vamos al Diseño de Experimentos.

**Comment by LauraCarbonell** [comment]
Revisando la información acerca de los efectos sobre el cambio climático parece no haber todavía estudios concluyentes ya que es difícil  pronosticar cuales serán estos efectos a largo plazo. Sin embargo, si parece estar demostrado la relación entre la disminución del numero de colonias de abejas y el aumento del **uso de plaguicidas** a través de diversas revistas científicas.

**Comment by marthapd** [comment]
Efectivamente, Laura. Me parece buena idea. Creo que es mejor centrarnos en el uso de plaguicidas. He encontrado algunos artículos al respecto.

**Comment by marthapd** [comment]
Breve revisión sobre la problemática del uso de plaguicidas neonicotinoides y su impacto en los polinizadores.

LAURA CARBONELL, MARTA PADIAL
Khemsearch Ciencia Abierta, Granada, España.

En la actualidad existe un creciente interés por el impacto que tienen determinados plaguicidas sobre las colonias de abejas ya que es de suma importancia el servicio que proporcionan a los ecosistemas naturales y a la agricultura. Por este motivo es necesaria su protección.Durante los años 2000-2010, el problema de la intoxicación por plaguicidas de las abejas parecía estar prácticamente resuelto. El número de casos disminuía en comparación con la década de los setenta. El problema de la intoxicación, sin embargo, no ha desaparecido, sino que se ha transformado de un problema de envenenamiento por pesticidas "tradicionales" como pesticidas organofosforados, a envenenamiento por nuevos plaguicidas como aquellos que contienen neonicotinoides como agente insecticida.[1] 

Aunque el papel de los insecticidas en la disminución global de las poblaciones de polinizadores sigue estando en entredicho, ahora es más evidente que algunos muestran efectos negativos en su salud, tanto a nivel individual como de colonia. [2]En menos de 20 años, los neonicotinoides se han convertido en una clase ampliamente utilizada de insecticidas con una cuota de mercado global de más del 25%. Para los polinizadores, esto ha transformado el paisaje agroquímico. Estos productos químicos imitan el neurotransmisor de acetilcolina y son altamente neurotóxicos para insectos. Su amplia aplicación, persistencia en suelo y agua y potencial de absorción por cultivos y plantas silvestres via xilema  hacen que los polinizadores estén expuestos a concentraciones subletales  de neonicotinoides durante la mayor parte del año.[3] 

Tomasz Kiljanek et al. revisaron los casos de intoxicación relacionados con neonicotinoides. Esto es de importancia por lo anteriormente comentado, ya que supone que existe una intoxicación secundaria inducida por la planta contaminada con plaguicidas y consumida continuamente por la misma generación de abejas y que puede causar una  intoxicación subletal. Esto conduce a un comportamiento anormal identificado durante la intoxicación aguda. El proceso descrito se refiere principalmente a colonias robustas y fuertes que fueron capaces de recolectar mucha comida debido a la efectiva protección de las plantas con plaguicidas. La habilidad descrita por los neonicotinoides para persistir en el tejido vegetal, en suelo y agua causa un riesgo extremo de toxicidad y por tanto llegar a provocar el colapso de la colonia. Esto se debe a que los neonicotinoides afectan al sistema nervioso central de la abeja produciendo efectos adversos en su capacidades motoras y de navegación por lo que no son capaces de recolectar alimento y volver a la colmena. La ausencia de un adecuado cumplimiento de las funciones sociales asignadas a las diferentes castas de abejas, conduce a la extinción de la colmena. [3] 


REFERENCIAS

[1] T. Kiljanek, A. Niewiadowska, and A. Posyniak, ͞Pesticide poisoning of honeybees: A review of symptoms, incident classification, and causes of poisoning,͟J. Apic. Sci., vol. 60, no. 2, pp. 5–24, 2016. 

[2] https://www.uv.es/uvweb/master-chemistry/en/blog/el-agroquimico-que-pone-en-peligro-a-las-abejas-1285949128883/GasetaRecerca.html?id=1285956888076

[3] J. P. Van der Sluijs, N. Simon-Delso, D. Goulson, L. Maxim, J. M. Bonmatin, and L. P. Belzunces, ͞Neonicotinoids, bee disorders and the sustainability of pollinator services,͟Curr. Opin. Environ. Sustain., vol. 5, no. 3–4, pp. 293–305, 2013.

**Comment by marthapd** [comment]
Revisa por favor, a ver si encuentras algo que sea necesario modificar. Gracias.

**Comment by LauraCarbonell** [comment]
He añadido algunos detalles pero creo que esta muy completo. Quizás podríamos añadir una  comparación entre como afecta el uso de plaguicidas a la muerte de colonias en los diferentes paises de la UE.

**Comment by marthapd** [comment]
Gracias, reviso y añado lo que me comentas.
